contrast.html

```diff
@@ -126,7 +126,7 @@ <h2 id="film" class="no-display">Video on Colors with Good Contrast</h2>
       <p>[ <a href="keyboard.html"><strong>Previous:</strong> Keyboard Compatibility</a> | <a href="Overview.html">Up: Overview</a> | <a href="layout.html"><strong>Next:</strong> Clear Layout and Design</a> ]</p>
 
       <h2 id="what">What is Colors with Good Contrast?</h2>
-      <p>There needs to be sufficient contrast between text color and its background (technically called <em>luminosity contrast ratio</em>). This includes text on images, icons, and buttons. Too little contrast makes the content difficult for some people to read. (While some people need high contrast, some people are sensitive to brightness and need to change the colors.)</p>
+      <p>There needs to be sufficient contrast between text color and its background (technically called <em>luminosity contrast ratio</em>). This includes text on images, icons, and buttons. Too little contrast makes the content difficult for some people to read. (While some people need high contrast, some people are sensitive to brightness and need to <a href="https://www.w3.org/WAI/perspectives/customizable.html">change the colors</a>.)</p>
```

Comment:
Adding cross-references can be a bit confusing for some people, especially since the link text is not used consistently.